### PR TITLE
jasper: Update to 2.0.33

### PR DESCRIPTION
**Pull request by evanmiller**
Closes: https://trac.macports.org/ticket/63300

#### Description

Remove now-obsolete patch, and point to the official release tarballs instead of the auto-generated GitHub tag tarballs.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.4.11 8S165 Power Macintosh
Component versions: DevToolsCore-798.0; DevToolsSupport-794.0

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: graphics/jasper/Portfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ PortSystem      1.0
 PortGroup       cmake 1.0
```
Comment (reneeotten) [inline graphics/jasper/Portfile:4]: if you change this to the `cmake 1.1` PG, you can remove the line `cmake.out_of_source yes` below

File: graphics/jasper/Portfile [hunk 2]
```diff
@@ -17,13 +18,12 @@ long_description \
 
 homepage        http://www.ece.uvic.ca/~mdadams/jasper/
 
-checksums       rmd160 291b4e628c2966b867b931224861920fc211da1b \
-                sha256 10da10708407c21214c2d9a8a39b3b42ef5c49325be2305975e3c68022d15d1c
+checksums       rmd160 ed5665914e6c8a312535bd771028f7bf6f88b542 \
+                sha256 28d28290cc2eaf70c8756d391ed8bcc8ab809a895b9a67ea6e89da23a611801a
```
Comment (reneeotten) [inline graphics/jasper/Portfile:22]: please add the `size` checksum as well

Comment (evanmiller) [reply graphics/jasper/Portfile]: Done

File: graphics/jasper/Portfile (hunk 1 above)
Comment (evanmiller) [reply graphics/jasper/Portfile]: Done

**Comment by reneeotten** [comment]
Closed by 80c0dd9667376b91d85a6ed071856350e367c4c3, which was committed to `master` by the maintainer. Thanks for your contribution @evanmiller - likely @jmroot didn't notice this PR when he updated the port!

**Comment by evanmiller** [comment]
@reneeotten Works for me!

**Pull request closed without merging**